SurveyApp.WebAPI/Controllers/SurveyController.cs

```diff
@@ -0,0 +1,172 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+using System.Web.Http.Cors;
+using AutoMapper;
+using SurveyApp.BLL.Infrastructure;
+using SurveyApp.BLL.Services.Interfaces;
+using SurveyApp.BLL.Models;
+using SurveyApp.WebAPI.Models;
+
+namespace SurveyApp.WebAPI.Controllers
+{
+    //[Authorize]
+    [RoutePrefix("api/survey")]
+    [EnableCors(origins: "*", headers: "*", methods: "*")]
+    public class SurveyController : ApiController
+    {
+        private const string MvcAccountControllerUri = "http://localhost:1169/Account/GetCurrentUserName";
+
+        public ISurveyService SurveyService { get; }
+        public IIdentityUserService UserService { get; }
+        public IMapper Mapper { get; }
+
+        public SurveyController(ISurveyService surveyService, IIdentityUserService userService, IMapper mapper)
+        {
+            SurveyService = surveyService;
+            UserService = userService;
+            Mapper = mapper;
+        }
+
+        [HttpGet]
+        [Route("")]
+        public HttpResponseMessage Get()
+        {
+            if (SurveyService != null)
+            {
+                return new HttpResponseMessage(HttpStatusCode.Created);
+            }
+            else
+            {
+                return new HttpResponseMessage(HttpStatusCode.Accepted);
+            }
+        }
+
+        [HttpGet]
+        [Route("auth")]
+        public HttpResponseMessage GetUser()
+        {
+            if (User != null)
+            {
+                return new HttpResponseMessage(HttpStatusCode.Created);
+            }
+            else
+            {
+                return new HttpResponseMessage(HttpStatusCode.Accepted);
+            }
+        }
+
+        [HttpGet]
+        [Route("getall/{templatesOnly:bool}")]
+        public IEnumerable<SurveyViewModel> GetAllSurveys(bool templatesOnly)
+        {
+            string currentUserId = RequestContext.Principal.Identity.Name;
+
+            IEnumerable<UploadSurveyServiceModel> surveysServiceModels = (templatesOnly)
+                ? SurveyService.GetSurveysTemplates(currentUserId)
+                : SurveyService.GetSurveysCreatedByUser(currentUserId);
+
+            IEnumerable<SurveyViewModel> surveysForUser = Mapper.Map<IEnumerable<SurveyViewModel>>(surveysServiceModels);
+
+            if (!surveysForUser.Any())
+            {
+                surveysForUser = new List<SurveyViewModel>();
+            }
+            return surveysForUser;
```

Comment:
Please use a blank line before 'return' statement. It makes a code more readable.